.travis.yml

```diff
@@ -35,19 +35,7 @@ notifications:
 
 # command to run tests
 script:
-  - pipenv run coverage run --source=imapfw,imapfw.py -p -m imapfw.edmp
-  - pipenv run coverage run --source=imapfw,imapfw.py -p -m imapfw.mmp.manager
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -h
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py noop
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -c multiprocessing unitTests
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -c threading unitTests
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all noop
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all testRascal
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all examine
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c multiprocessing syncAccounts -a AccountA
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA -a AccountA -a AccountA
-  - pipenv run coverage combine .coverage*
+  - make test
```

Comment:
I'm not sure this is the same. What happens with the makefile when one test fails? Are the others run?